docs/splits.md

```diff
@@ -125,7 +125,7 @@ Example:
 
 ```py
 # Records 0 (included) to 99 (excluded).
-tfds.load("mnist", split="test[:99%]", rounding="pct1_dropremainder")
```

Comment:
Why changing this ?

Reply:
Please see #1740 tfds.load() have no param rounding